CHANGELOG.next.asciidoc

```diff
@@ -136,6 +136,7 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 - Enable early event encoding in the Elasticsearch output, improving cpu and memory use {pull}38572[38572]
 - The environment variable `BEATS_ADD_CLOUD_METADATA_PROVIDERS` overrides configured/default `add_cloud_metadata` providers {pull}38669[38669]
 - Update to Go 1.22.7. {pull}41018[41018]
+- Update container install docs to remove setup container automatically on completion. {pull}41069[41069]
```

Comment:
```suggestion
```

We don't need changelog entries for docs update, the changelog goal is to inform users of new features/changes they need to be aware when updating.